R/summarize_weighted_production.R

```diff
@@ -1,50 +1,101 @@
-#' Summarize production based on the weight of each loan per sector per year
+#' Summaries based on the weight of each loan per sector per year
 #'
-#' Summarize production based on the weight of each loan per sector per year.
+#' Based on the based on the weight of each loan per sector per year,
+#' `summarize_weighted_production()` and `summarize_weighted_buildout()`
+#' summarize the production and build-out, respectively.
```

Comment:
I placed both summaries in a single file, and a single helpfile. They seem too closely related to be separate.